index.js

```diff
@@ -863,6 +884,13 @@
     return f(z, this.value);
   });
 
+  //  Just#sequence :: Applicative f => Maybe (f a) ~> (b -> f b) -> f (Maybe a)
+  Just.prototype.sequence =
+    def('Just#sequence', [Function], function(of) {
+      void of;
```

Comment:
`make lint` passes for me without this line, thanks to #31. Was it failing for you and/or CircleCI?